app/controllers/admin/services_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Admin
+
+  class ServicesController < ApplicationController
+
+    def cancel_reservations_for_offline_instruments
+      InstrumentOfflineReservationCanceler.new.cancel!
+      render nothing: true
```

Comment:
I'm assuming we just want the 500, so we don't need to worry about the error case, correct?